src/app/GlobalStyles.tsx

```diff
@@ -7,16 +7,31 @@ const GlobalStyles = createGlobalStyle`
 
     body {
   margin: 0;
+  
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  color: var( --color-dark-green);
+    background-color: var( --color-background-light)
+}
+:root {
+  --color-background-light: #f6f5f2;
+  --color-background-dark: #d1e1db;
+  --color-gold: #ceb372;
+  --color-yellow: #ffe4a0;
+  --color-choral: #f47851;
+  --color-dark-green: #2b5113;
```

Comment:
Maybe you can apply the same naming structure for the green color, by saying --color-green-dark. Additionally, you might encounter a problem during your project, when you want to use another dark green color. To avoid that, you can give the different shapes of green a number: e.g. --color-green-700 (a dark green) --color-green-500 (normal green) & color-green-300 (light-green). This also allows you to add new shapes of green as well at any moment during your project 🎨

Reply:
good idear, thank you!